torch/csrc/jit/codegen/fuser/cuda/fused_kernel.cpp

```diff
@@ -64,6 +65,8 @@ void getMajorMinor(const cudaDeviceProp* const prop, int& major, int& minor) {
     major = 8;
     minor = 0;
   }
+
+  return (major != prop->major) || (minor != prop->minor);
```

Comment:
So for 11.1 this is always `false`, major == prop->major, same for minor. This means here https://github.com/pytorch/pytorch/pull/50319/files#diff-5e44e2a0e9a686349e4152cd3f1255ae55c3c208b22112f194144d8c6612f61aR296 we are going to ptx - but it was supposed to be the other way around?

Reply:
Good catch.